crates/holochain/src/fixt.rs

```diff
@@ -510,87 +508,3 @@ fixturator!(
     CallContext;
     constructor fn new(Zome, FunctionName, HostContext, InvocationAuth);
 );
-
-fixturator!(
```

Comment:
Was this just unused?

Reply:
Yes, it was only used in one place and in a funny way where first an invocation was fixturated, then the properties were changed and then it was converted to a `ZomeCall` struct.